### ISSUE TITLE
Flaky E2E: Media: Edit: click to edit selected image

### ISSUE THREAD

**Issue by worldomonation** [Automattic/wp-calypso#56254]
### Spec

wp-media__edit-spec.ts

### Suite

Click to edit selected image

### Framework

Playwright

### Logs

```
page.click: Timeout 30000ms exceeded.
=========================== logs ===========================
waiting for selector ".editor-media-modal-detail__edition-bar .editor-media-modal-detail__edit:visible"
  selector resolved to visible <button disabled type="button" class="button editor-medi…>…</button>
attempting click action
  waiting for element to be visible, enabled and stable
element is not enabled - waiting...
============================================================
at MediaPage.editImage (/home/teamcity-1/buildAgent/work/c4a9d5b38c1dacde/packages/calypso-e2e/src/lib/pages/media-page.ts:100:19)
at Object.<anonymous> (/home/teamcity-1/buildAgent/work/c4a9d5b38c1dacde/test/e2e/specs/specs-playwright/wp-media__edit-spec.ts:56:4)
```

**Comment by worldomonation** [comment]
This is likely because the upload isn't finished before the edit image button is clicked.

**Comment by worldomonation** [comment]
As noted in #56284 - 

The changes there ended up introducing a permafail on mobile. Looking into it.

**Comment by worldomonation** [comment]
Fixed by #56414.
